src/character.cpp

```diff
@@ -12284,6 +12284,11 @@ int Character::run_cost( int base_cost, bool diag ) const
     if( !is_mounted() ) {
         if( movecost > 105 ) {
             movecost *= mutation_value( "movecost_obstacle_modifier" );
+
+            if( has_proficiency( proficiency_prof_parkour ) ) {
+                movecost *= .5;
+            }
+
```

Comment:
@Pseudogeist Previously the `mutation_value` section specified the movecost - it is specifiable in JSON for any mutation.
Now this if statement with the hardcoded profession id does.

Reply:
Ahhhh, right, sorry, I misunderstood. Hmm. I'll have to look into how all of that is calculated and when, and how the existing pathfinding code handles it all, this is some good food for thought. Thanks!